_data/simple-icons.json

```diff
@@ -776,7 +776,7 @@
         {
             "title": "Arduino",
             "hex": "00979D",
-            "source": "https://cdn.arduino.cc/projecthub/img/Arduino-logo.svg"
+            "source": "https://www.arduino.cc/"
```

Comment:
According to their [guidelines](https://www.arduino.cc/en/trademark), I wonder if we can keep their logo as-is. We might need to use their [Community logo](https://www.arduino.cc/en/Trademark/CommunityLogo)... What do you think?

Reply:
This sentence and the "You may" section, but we might be ok. Resolving this comment!

Reply:
Can you also add their guidelines btw:
https://www.arduino.cc/en/trademark